app/Ptr/PtrControllerTest.php

```diff
@@ -169,7 +186,16 @@ private function endEntityRange(): string {
   }
 
   protected function canCreateInsideEntityRange(): JsonResponse {
+    $this->dns
```

Comment:
this mocking is pretty redundant, can we DRY this up with a private function in this class?